events/api.py

```diff
@@ -2001,6 +2008,46 @@ class Meta:
         exclude = ["id", "event"]
 
 
+# Simplified RegistrationSerializer to avoid circular imports
```

Comment:
This worries me as well. 

The circular import was caused because `EventSerializer` uses `RegistrationSerializer` for `Registration` and `RegistrationSerializer` uses `EventSerializer` for `Event`. Circular imports happen when they are in different files.

I tried to move LinkedEventsSerializer to separate file but end result was that I had to move basically all the Event related serializer + `RegistrationSerializer`  as well. One of the problematic lines was `self.related_serializer = globals().get(self.related_serializer, None)` in `JSONLDRelatedField` which is used to get serializer if it's string. It expects Serializers to be in same file. 

I tried to improve this little bit be adding `RegistrationBaseSerializer` in `registration/serializers` and inherit it for the `RegistrationSerializer` which is now only serializer for Registration. `RegistrationBaseSerializer` container all the fields except event and `RegistrationSerializer` contains event field, methods to mutate Registration and validation. It's not perfect but at least after that we don't have to maintain 2 registration serializers.

Reply:
This will do nicely, thank you!